website/src/segment.tsx

```diff
@@ -190,16 +190,15 @@ const MorphemicSegmentation = (p: {
   onOpenDetails: Props["onOpenDetails"]
   level: ViewMode
 }) => {
-  // If there is no segmentation, return two line breaks for the
+  // If there is no segmentation, return a hard break for the
   // morphemic segmentation and morpheme gloss layers.
   if (!p.segments?.length) {
     return (
       <>
-        <br />
-        <br />
+        <hr className={css.fillerLine}/>
```

Comment:
I think we should make a component for this `<hr/>` tag. That might look like:

```ts
const FillerLine = () => (<hr className={css.fillerLine}/>)
```

And then we also want to retain the vertical rhythm of our word annotations, so make sure to include two here, not just one. i.e.

```ts
<FillerLine />
<FillerLine />
```